const.go

```diff
@@ -0,0 +1,21 @@
+//go:build !386 && !arm
+
+// Copyright 2019 The nutsdb Author. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nutsdb
+
+import "math"
+
+const MAX_SIZE = math.MaxInt32
```

Comment:
PR description says that this should be `MaxUInt32`, but it is only `MaxInt32`. Is this a mistake?

Previously it was `MaxUInt32` and I think the whole point of making it a constant was that 64bit targets could keep `MaxUInt32` instead of just changing it `MaxInt32` in `tx.go`.

Reply:
You're right, this looks like a mistake.

Reply:
I made a fix in #575.